### ISSUE TITLE
We need a quick-start guide. 

### ISSUE THREAD

**Issue by iandobbie** [MicronOxford/cockpit#566]
Chris thinks he has cockpit installed and has no idea what to do. We need a quick-start guide. Once installed how do you run cockpit, what do the various windows do? suggested workflows etc...

**Comment by iandobbie** [comment]
I have started to write this, but how are we outputting the .rst files into some formatted version. HTML or pdf?

**Comment by carandraug** [comment]
> I have started to write this, but how are we outputting the .rst files into some formatted version. HTML or pdf?

The output in html can be generated by running `python3 setup.py build_sphinx`.

**Comment by carandraug** [comment]
As part of the quick start guide we spoke about install instructions. I have written the Debian specific instructions on my "wip-install-linux" branch (or see commit [257e6eda](https://github.com/carandraug/cockpit/commit/257e6eda7c30f159537a6967220b25674103e4fe))

**Comment by iandobbie** [comment]
I just wrote similar instructions for a blank macos catalina install. I put it directly in the INSTALL file. We also need to deal with the link format as the github readme display wont follow them. My updates are in https://github.com/iandobbie/cockpit/tree/macos-install

**Comment by iandobbie** [comment]
My machine didn't have sphinx installed so couldn't build the docs. Do we need to add it to the requirements? I have pulled both your linux instructions and the macos install instructions into the master branch. Needs a bit more editing to make the format correct.

**Comment by carandraug** [comment]
> My machine didn't have sphinx installed so couldn't build the docs. Do we need to add it to the requirements?

No, because sphinx is only required to build the documentation which the vast majority of people won't do. We do have a check for its presence on microscope's setup.py which gives a better error message. I can pull something like that for cockpit, what do you think?
